core/modules/user/templates/user-simplified-page.tpl.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @file
+ * Theme template for the "Log in", "Reset password", and "Create new Account"
+ * forms when shown on simplified (layout suppressed) pages.
+ *
+ * Available variables:
+ *   - $form: The form render array.
+ *   - $links: A render array with related form links.
+ *
+ * @since 1.30.0 Template added
+ * @ingroup themeable
+ */
+?>
+
+<div class="<?php print implode(' ', $classes); ?>">
+  <div class="user-simplified-page-wrapper">
+    <div class="user-simplified-page-wrapper-inner">
+      <?php if ($logo_image): ?>
+        <div class="user-simplified-page-logo">
+          <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home" class="logo">
+            <?php print $logo_image; ?>
+          </a>
+        </div>
+      <?php endif; ?>
+      <h1 class="site-name">
+        <a href="<?php print $front_page; ?>" title="<?php print t('Home'); ?>" rel="home"><span><?php print $site_name; ?></span></a>
+      </h1>
+      <h2 class="user-simplified-page-title">
+        <?php print $page_title; ?>
+      </h2>
+      <div class="user-simplified-page-messages">
+        <?php print $messages; ?>
+      </div>
+
+      <?php print render($form); ?>
+      <div class="user-tabs-links">
```

Comment:
This could use a better class name since these aren't tabs. I get the idea was that they _were tabs_ before. But maybe we just match the other prefixes with:
```suggestion
      <div class="user-simplified-page-links">
```

Reply:
To make them consistent with the "links" theming, I also called this "user-form-links".